drivers/flash/soc_flash_nrf.c

```diff
@@ -127,8 +139,33 @@ static void nvmc_wait_ready(void)
 	}
 }
 
-static int flash_nrf_read(struct device *dev, off_t addr,
-			    void *data, size_t len)
+enum nvmc_config {
```

Comment:
by default enums are int size, but it possible to switch to other size.
Fro the other hand nice to use this in func. declaration.
Maybe no worth to be so carefully.